docker_images/common/tests/test_api.py

```diff
@@ -26,22 +26,17 @@
     "speech-segmentation",
     "tabular-classification",
     "tabular-regression",
-    "text-classification",
     "text-to-image",
     "text-to-speech",
     "token-classification",
     "conversational",
     "feature-extraction",
-    "question-answering",
     "sentence-similarity",
     "fill-mask",
     "table-question-answering",
     "summarization",
     "text2text-generation",
     "text-classification",
-    "text-to-image",
-    "text-to-speech",
-    "token-classification",
```

Comment:
* The removed ones should be reverted
* We should also add `image-to-image` here

Reply:
They are duplicates "text-to-image" exists in there already. Cleaning up ;-)